testing/test_python.py

```diff
@@ -0,0 +1,28 @@
+import pytest
```

Comment:
Oh my, how did I miss that. :sweat_smile: 

Thanks, just updated the appropriate test.